proof/crefine/X64/Machine_C.thy

```diff
@@ -95,6 +78,26 @@ assumes invalidateLocalPageStructureCacheASID_ccorres:
            (doMachineOp (invalidateLocalPageStructureCacheASID vspace asid))
            (Call invalidateLocalPageStructureCacheASID_'proc)"
 
+
+(* FPU-related machine ops *)
+
+assumes saveFpuState_ccorres:
```

Comment:
Much nicer machine interface than what we had before. Glad it worked out so well.